### PR TITLE
fix(wallet): Update ENS Offcahin Lookup Message

### PR DESCRIPTION
**Pull request by Douglashdaniel**
## Description 
Updates the `ENS Offchain Lookup` message to say `It looks like you've entered an ENS address. We'll need to use a third-party resolver to resolve this request, which may be able to see your IP address and domain.`

Resolves <https://github.com/brave/brave-browser/issues/27495>

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [x] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [x] New files have MPL-2.0 license header
- [x] Adequate test coverage exists to prevent regressions
- [x] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [x] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

Before:

Screenshot 2022-12-21 at 12 27 10 PM

After:

Screenshot 2022-12-21 at 12 26 12 PM

### CODE REVIEW

File: components/brave_wallet_ui/stories/locale.ts [hunk 1]
```diff
@@ -100,7 +100,7 @@ provideStrings({
   braveWalletBuyTapBuyNotSupportedMessage: 'Buy not supported for selected network',
   braveWalletSearchingForDomain: 'Searching for domain...',
   braveWalletEnsOffChainLookupTitle: 'Brave supports using off-chain gateways to resolve .eth domains.',
-  braveWalletEnsOffChainLookupDescription: 'It looks like youve entered an ENS address. Well need to use a third-party resolver to resolve this request. This helps ensure your .eth domain isnt leaked, and that your transaction is secure.',
+  braveWalletEnsOffChainLookupDescription: 'It looks like youve entered an ENS address. Well need to use a third-party resolver to resolve this request, which may be able to see your IP address and domain.',
```
Comment (josheleonard) [inline components/brave_wallet_ui/stories/locale.ts:103]:
```suggestion
  braveWalletEnsOffChainLookupDescription: 'It looks like you\'ve entered an ENS address. We\'ll need to use a third-party resolver to resolve this request, which may be able to see your IP address and domain.',

```

Comment (josheleonard) [reply components/brave_wallet_ui/stories/locale.ts:103]: Missing apostrophes

**Review by josheleonard — CHANGES_REQUESTED** [review]
Missing apostrophes

File: components/brave_wallet_ui/stories/locale.ts (hunk 1 above)
Comment (Douglashdaniel) [reply components/brave_wallet_ui/stories/locale.ts]: Fixed!

**Review by josheleonard — APPROVED** [review]
LGTM

**Pull request merged**